### PR TITLE
LB Tweaks

### PR DESCRIPTION
**Pull request by DavidIHunt**
Please don't merge this just yet.

### CODE REVIEW

File: server/api/controllers/config/upstreams/upstreamsConfigController.js [hunk 1]
```diff
@@ -31,10 +34,13 @@ function postUpstreamsConfig(req, res, next) {
   let key = body.key;
   let environmentName = body.Value.EnvironmentName;
 
+  body.UpStreamKeepalives = 32;
```
Comment (filso) [inline server/api/controllers/config/upstreams/upstreamsConfigController.js]: that looks a bit random. we want to always overwrite Keepalives to 32 no matter what users input?

File: server/api/controllers/config/upstreams/upstreamsConfigController.js [hunk 2]
```diff
@@ -47,10 +53,13 @@ function putUpstreamConfigByName(req, res, next) {
   let user = req.user;
   let environmentName = body.EnvironmentName;
 
+  body.UpStreamKeepalives = 32;
```
Comment (filso) [inline server/api/controllers/config/upstreams/upstreamsConfigController.js]: same here

File: server/api/controllers/config/upstreams/upstreamsConfigController.js [hunk 3]
```diff
@@ -19,7 +19,10 @@ function getUpstreamsConfig(req, res, next) {
 function getUpstreamConfigByName(req, res, next) {
   let key = req.swagger.params.name.value;
   let accountName = req.swagger.params.account.value;
-  return dynamoHelper.getByKey(key, { accountName }).then(data => res.json(data)).catch(next);
+  return dynamoHelper.getByKey(key, { accountName }).then(data => {
+    data.Value.UpStreamKeepalives = 32;
```
Comment (filso) [inline server/api/controllers/config/upstreams/upstreamsConfigController.js]: so basically we say it's always 32?

**Comment by DavidIHunt** [comment]
@filso - Yeah this one is a bit weird.. The value needs to be 32 apparently. I don't know if people will be reading this value directly from the database so I set it every time the record updates. I know that some records won't have a value, so I make sure it's 32 when anyone reads. Finally I take into account the some people might be using the API to update records which would bypass the disabled UI field, so I set it on updates as well. I'm open to suggestions. @duncanhall @Merlin-Taylor .. What do you guys think?

**Pull request merged**